mLRS/CommonTx/crsf_interface_tx.h

```diff
@@ -172,7 +172,7 @@ bool tTxCrsf::transmit_start(void)
 
 // a frame is sent every 4 ms, frame length is max 64 bytes
 // a byte is 25 us
-// gaps between frames are 1 ms or so
+// gaps between frames are 3 ms or so
```

Comment:
the comment is true?
with 4 ms slices and 2x64 bytes at 400000 where can't be 3 ms space in general. Is this referring to the time from end of a received frame to start of the next receiving frame? If so maybe ... // gaps between incomming frames ....

Reply:
The original comment seemed wrong to me.  The gaps between send and receive are ~0 or ~3 depending on if you mean receive -> send or send->receive.  There were no 1ms gaps to be seen on the scope.  How about "The gaps between received frames are at least 2.4 ms"

Reply:
I think the fact you don't see 1ms gaps on the scope is ONLY because there are only short messages!
as said, at 400000 2x64 bytes are 3.2 ms which need to fit in the 4 ms slot, so technically it are only 0.8 ms !!